Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at
  http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
diff --git a/pom.xml b/pom.xml
index b8c0d9a9d..7c2674e63 100755
--- a/pom.xml
+++ b/pom.xml
@@ -920,5 +920,12 @@
       </plugin>
     </plugins>
   </reporting>
-
+
+  <distributionManagement>
+    <repository>
+      <id>jfrog-linkedin</id>
+      <name>linkedin-releases</name>
+      <url>https://linkedin.jfrog.io/artifactory/zookeeper</url>
+    </repository>
+  </distributionManagement>
 </project>
